docs/usage/from-a-terminal.rst

```diff
@@ -1130,7 +1117,7 @@ Global overview
         -h, --host            Activates or disables the generation of the
                                 hosts file(s).
                                 Configured value: True
-        -ip CLI_TESTING__HOSTS_IP, --hosts-ip CLI_TESTING__HOSTS_IP
+        -ip CLI_TESTING__HOSTS_IP, --host-ip CLI_TESTING__HOSTS_IP
```

Comment:
It is called `hosts` file 😉 

But I'm open to adding the singular version. Can you please add/modify it in the resource code too ? 

https://github.com/funilrys/PyFunceble/blob/2b12f88e69d549e52de681c1fd490c6f4c54ec04/PyFunceble/cli/entry_points/pyfunceble/cli.py#L647

Reply:
It was and find/replace error

Should be fixed